cloudferrylib/scheduler/scheduler.py

```diff
@@ -14,11 +14,11 @@
 
 
 import multiprocessing
-import traceback
 
 from cloudferrylib.scheduler.namespace import Namespace, CHILDREN
 from cloudferrylib.utils import utils
 from cursor import Cursor
+from signal_handler import IgnoreInterruptHandler, InterruptHandler
```

Comment:
Due to OpenStack Style Guidelines, I would suggest do not import objects, only modules.
(http://docs.openstack.org/developer/hacking/#imports)